### DISCUSSION TITLE
Deconz LXC doesn't assign a device to deconz, and it's nowhere mentioned how to assign a device.

### DISCUSSION THREAD

**Discussion by PascalHaury** [tteck/Proxmox#1637] [General]
The script for the Deconz lxc is working as intended, and the USB passthrough is also set correctly.

But you have to set device manually to deconz, because it is not automatically done! At the Moment no device is assigned to the deconz default install.

Here are my steps, to assign the conbee stick to the Deconz LXC:

1. Get the device name:
`ls -l /dev/serial/by-id/`

2. Set assign the device to deconz in:
`/etc/systemd/system/multi-user.target.wants/deconz.service`
2.1 Set the Device:
in my case 
`ExecStart=/usr/bin/deCONZ -platform minimal --http-port=80 --dev=/dev/ttyACM1`
**The --dev entry is not automatically set by the script, and it is nowhere mentioned.** 

![image](https://github.com/tteck/Proxmox/assets/32424321/a4943f15-f4b6-4622-836d-3dfccfa9d851)

3. Run 
`systemctl daemon-reload`

4. Restart Deconz
`systemctl restart deconz`

![image](https://github.com/tteck/Proxmox/assets/32424321/2ccfb773-3e9d-4833-83df-f6c31e933f63)

Otherwise deconz does not see the connected usb stick, although the USB passthrough is working

To reproduce the issue, simply create a new deconz lxc. 
Doesn't matter if a zigbee stick is connected or not.

Maybe it is possible to add a tutorial to the script, how to assign a device to deconz.

**Comment by tteck** [comment]
Or, just link to your post. 😉
